nix/pkgs/default.nix

```
@@ -99,7 +99,10 @@ let
 
   # not available in 20.03 and we depend on several recent changes
   # including stylish-haskell support
-  nix-pre-commit-hooks = import (sources."pre-commit-hooks.nix");
+  nix-pre-commit-hooks = (pkgs.callPackage ((sources."pre-commit-hooks.nix") + "/nix/default.nix") {
+    system = builtins.currentSystem;
+    inherit sources;
```

```suggestion
    inherit (sources) nixpkgs;
```

If we just pass in `sources` we're relying on the fact that they pull `nixpkgs` out of that and that's what ours is called. This way we explicitly use the `nixpkgs` argument.